code/modules/research/xenobiology/crossbreeding/_potions.dm

```diff
@@ -73,7 +73,7 @@ Slimecrossing Potions
 	icon = 'icons/roguetown/items/cooking.dmi'
 	icon_state = "lovebottle"
 
-/obj/item/slimepotion/lovepotion/attack(mob/living/M, mob/user)
```

Comment:
No? This handles only living things. What you declare here is merely a suggestion.